docs/snippets/change-a-currency-symbol.md

```diff
@@ -0,0 +1,28 @@
+# Rename a country
+
+See the [currency list](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-core-functions.html#source-view.475) for reference on currency codes.
+
```

Comment:
Totally optional, but could remove one of these extra line breaks?

Reply:
Good catch, @barryhughes. I removed the obsolete line breaks.